draft-irtf-cfrg-opaque.md

```diff
@@ -2264,6 +2264,19 @@ attacker from using this data to mount a server spoofing attack. Supporting impl
 need to consider allowing separate AKE and OPRF algorithms in cases where the HSM is
 incompatible with the OPRF algorithm.
 
+## Client Authentication Using Credentials
+
+For scenarios in which the client has access to private state that can be persisted across
+registration and login, the client can back up the `randomized_password` variable (as
+computed in {{finalize-request}}) so that upon a future login attempt, the client can
+authenticate to the server by using `randomized_password` instead of the original password,
+and supplying an arbitrary password as input to `CreateCredentialRequest` in the login
+phase. This provides an advantage over the regular authentication flow for login
```

Comment:
I suggest we add that we give all the details on how to do it, adding that the OPRF finish/output and stretching can be skipped in favor of directly using the randomized henceforth. What do you think?

Reply:
Good idea. I added a little bit more to the description to clarify how this would be done. Let me know what you think!